pkg/loadbalance/weighted_random_with_alias_method_test.go

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 CloudWeGo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package loadbalance
+
+import (
+	"fmt"
+	"testing"
+)
+
```

Comment:
Where is benchmark codes for `Next` ?

Reply:
It actually had been contained in `pkg/loadbalance/weighted_balancer_test.go` , and the result pic is in the describe of this PR.
 
I can add it benchmark in `pkg/loadbalance/weighted_random_with_alias_method_test.go` if is needed.